_includes/salary-tracker.html

```diff
@@ -0,0 +1,417 @@
+            var monthlyHighEle = document.getElementById('salary-monthly-amount-high');
+
+            if (countryArr[0].benefits !== undefined) {
+                var splitBenefits = countryArr[0].benefits.split(' | ').join('\n');
+                benefitsEle.innerText = splitBenefits;
+            } else {
+                benefitsEle.innerText = "";
+            }
+
+            if (selectedPosition === null) {
+                if (countryArr[0].positions[0] !== undefined) {
+                    countryArr[0].positions.forEach(position => {
+                        var opt = document.createElement('option');
+                        opt.value = position.position;
+                        opt.innerHTML = position.position;
+                        selectRoleEle.appendChild(opt);
+                    });
+                    selectItemByValue(document.getElementById('salary-tracker-role-picker'), null);
+                    try {
+                        annualLowEle.innerText = countryArr[0].positions[0].annualLow;
+                        annualHighEle.innerText = countryArr[0].positions[0].annualHigh;
+                        monthlyLowEle.innerText = countryArr[0].positions[0].monthlyLow;
+                        monthlyHighEle.innerText = countryArr[0].positions[0].monthlyHigh;
+                    } catch (err) {
+                        annualLowEle.innerText = 0;
+                        annualHighEle.innerText = 0;
+                        monthlyLowEle.innerText = 0;
+                        monthlyHighEle.innerText = 0;
+                    }
+                }
+            } else {
+                var positionArr = countryArr[0].positions.find(function (el) {
+                    return el.position === selectedPosition;
+                });
+                if (positionArr) {
+                    try {
+                        annualLowEle.innerText = positionArr.annualLow;
+                        annualHighEle.innerText = positionArr.annualHigh;
+                        monthlyLowEle.innerText = positionArr.monthlyLow;
+                        monthlyHighEle.innerText = positionArr.monthlyHigh;
+                    } catch (err) {
+                        annualLowEle.innerText = 0;
+                        annualHighEle.innerText = 0;
+                        monthlyLowEle.innerText = 0;
+                        monthlyHighEle.innerText = 0;
+                    }
+                }
+            }
+        }
+    }
+
+    // Main launch function
+    // Define the doc url and execute fetch promise
+    // Parse the returned data into a formatted/friendly array
+    // Remove the 'no positions loaded' selection option for positions
+    // Update the data
+    function main() {
+
+        var url = "https://docs.google.com/spreadsheets/d/e/2PACX-1vSr_TpN5LWs9ejhbmttNHfqG25LLi9rPKHPE3wYvovXwNNXSxcFsDSEBSEsEf-9MNfwFTKjLPh1FTLL/pub?output=csv";
+        var processedData = [];
```

Comment:
Should url be a configurable property?

Reply:
We should not really need to change the url as the sheet shouldn't change. And I'd prefer not to leave it change-able from within Site Leaf.

What are your thoughts as to why it should be configurable? Perhaps I may not be missing a use-case

Reply:
I had initially thought It is something that may change, since you have clarified that. It looks good to me 👍🏾